cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,40 @@
 /// <reference types='cypress' />
+let user;
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
-  it('', () => {
+  it('can register a new user', () => {
+    cy.viewport(1920, 1080);
+    cy.visit('https://demoqa.com/automation-practice-form');
+    cy.findByPlaceholder('First Name').type(user.firstName);
+    cy.findByPlaceholder('Last Name').type(user.lastName);
+    cy.get('#userEmail').type(user.email);
+    cy.contains('.custom-control-label', user.genders).click();
+    cy.findByPlaceholder('Mobile Number').type(user.phoneNumber);
+    cy.get('#dateOfBirthInput').type('{selectAll}12 jan 2003{enter}');
+    cy.get('#subjectsInput').type('a{upArrow}{enter}');
+    cy.contains('.custom-control-label', user.hobbies).click();
+    cy.findByPlaceholder('Current Address').type(user.address);
+    cy.get('#react-select-3-input').type('a{upArrow}{enter}');
```

Comment:
![image](https://github.com/mate-academy/qa_cypress_e2e_student_registration_form/assets/85580286/76c1d650-793f-441d-8243-1daac9faf6b7)
your test always fails on this step